docs/rules/require-valid-css-selector-for-testing-helpers.md

```diff
@@ -0,0 +1,33 @@
+# require-valid-css-selector-for-testing-helpers
+
+:wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+Test helpers and querySelector methods should be called with valid CSS selectors. Most of the time invalid selectors will result in a failing test but that is not always the case.
+
+One example of invalid CSS selectors which do not cause failing tests is when using unclosed attribute selector blocks. Tests happen to pass today in this case due to a quirk in the CSS selector spec which is marked as [WontFix by Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=460399#c6).
+
+## Rule Details
+
+This rule requires the use of valid CSS selectors in test helpers and querySelector methods.
+
+## Examples
```

Comment:
Can you update the examples to show `this.element.querySelector`/`this.element.querySelectorAll`/various `@ember/test-helpers` helpers?

Reply:
Done